### PR TITLE
OP-1185: Update inventory

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OP-1185.

### CODE REVIEW

File: sql/step_a106_alter_table_inventoryrow.sql
Comment (mwithi) [inline sql/step_a106_alter_table_inventoryrow.sql:1]: please modify directly `step_a104` by the moment it has not been release yet. thanks

Comment (ArnaudFonzam) [reply sql/step_a106_alter_table_inventoryrow.sql:1]: okay

File: .factorypath
Comment (mwithi) [inline .factorypath:1]: please remove this file and list it into your .gitignore

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**